Skeleton chat

Jenfer93 opened the pull request:
What's been added: 

- [x] Skeleton with default styling from the theme-skeleton
- [x] Styling to the Molly chat
- [x] Some minor changes to the molly chat to make close button and scroll to view to work. 

We should have a look at the themes, I'm not quite sure how you got the red to be primary color, as mine is very bright green! 
Please have a look at it, and let me know, and I will make the changes necessary :)

Melisa-Zorraindo commented on packages/molly/src/lib/components/Molly.svelte:
```diff
@@ -0,0 +1,89 @@
+							<MollyMessage type="assistant" message={answer} />
+						{/if}
+						{#if loading}
+							<MollyMessage type="assistant" message="Thinking.." />
```
```suggestion
							<MollyMessage type="assistant" message="Thinking..." />
```

Just a typo

Melisa-Zorraindo reviewed (approved):
I added a small styling suggestion, see if you like it.

Jenfer93 commented:
Thank you @Melisa-Zorraindo for the tips, I've added your changes, but I also hated the darker grey against the red, so I changed the slate darkness a tad, so please have a look at my changes again, I also added a little bit of drop shadow, just to make it pop a bit, and added a border, all this can be removed if you don't like it, just got veery flat to me so thought I would spice it up a bit
